src/points/slice.ts

```diff
@@ -159,4 +159,6 @@ export const {
 // attach points event data to.
 export const trackPointsEvent = createAction<PointsEvent>('points/trackPointsEvent')
 
+export const pointsDataRefreshStarted = createAction('points/pointsDataRefreshStarted')
```

Comment:
nit: should we name it `refreshPointsData` instead?

Reply:
@jeanregisser here i was following the pattern already established in that module with `tackPointsEvent` being a single exception 😅. and i believe it is following the [Redux Toolkit recommendation](https://redux-toolkit.js.org/usage/migrating-to-modern-redux#reducers-and-actions-with-createslice):
> Ideally, the naming of the reducers and actions should use the past tense and describe "a thing that happened", rather than an imperative "do this thing now", such as todoAdded instead of ADD_TODO

having that said I have no preference between using past tense or an imperative, just trying to be consistent. please let me know if you are feeling strong against the past tense. otherwise, I prefer to keep it for now.